fdb-extensions/src/test/java/com/apple/foundationdb/async/RangeSetTest.java

```diff
@@ -430,4 +541,78 @@ public void missingRanges() {
         }
     }
 
+    private static class MultipleTransactions implements AutoCloseable, Iterable<Transaction> {
```

Comment:
It doesn't have to be done in this PR, but I feel like this class would be pretty generally useful in other tests, so perhaps worth pulling out.

Reply:
Yeah, that makes sense. I've extracted the class from outside of the `RangeSetTest` into its own top-level class. I haven't migrated any other tests to use it, but now it's available.